example/ios/example/AppDelegate.mm

```diff
@@ -41,4 +46,15 @@ - (BOOL)concurrentRootEnabled
   return true;
 }
 
+// Push notifications registration
+- (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
+  [RNNotifications didRegisterForRemoteNotificationsWithDeviceToken:deviceToken];
+}
+- (void)application:(UIApplication *)application didFailToRegisterForRemoteNotificationsWithError:(NSError *)error {
+  [RNNotifications didFailToRegisterForRemoteNotificationsWithError:error];
+}
+- (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)userInfo fetchCompletionHandler:(void (^)(UIBackgroundFetchResult result))completionHandler {
+  [RNNotifications didReceiveBackgroundNotification:userInfo withCompletionHandler:completionHandler];
+}
```

Comment:
Cool - and similar to the native changes needed for react-native-app-auth, we can point folks to https://wix.github.io/react-native-notifications/docs/getting-started/.  Super long-term, we might encapsulate all of these kinds of things in our own native module, but I like this approach as our first full pass using these libs.

Reply:
Speaking of ☝️ should the Android changes be in this PR?  If/when we get there, can you provide a video of what the Android side looks like alongside iOS?

Reply:
Yes, I will have a separate PR for Android along with a video.